README.md

```diff
@@ -181,7 +181,8 @@ Command Option Names | Purpose
   [--gsi=name:hkey{field_name}[,rkey{field_name},proj_type{ALL\|KEYS_ONLY\|INCLUDE}]...]  | Allows for the declaration of secondary indexes
   [--required=field1...]                                                                | A list of attributes that are required for an instance of the model
   [--length-validations=field1:MIN-MAX...]                                              | Validations on the length of attributes in a model
-  
+  [--table-name=name] | Sets the name of the table in DynamoDB
```

Comment:
I would document how this is optional. Maybe `Sets the name of the table in DynamoDB, if different than the model name.`